downstream/modules/analytics/proc-create-groups.adoc

```diff
@@ -11,19 +11,19 @@ You can create and assign roles to a group that enables users to access specific
 .Procedure
 
 . Log in to cloud.redhat.com
-. Click image:images/cog.png[] > *Settings*.
-. From the sidebar, navigate to *User Access* > *Groups*.
-. Click *Create group*.
-. A setup wizard will now appear. Enter a group name and description, then click *Next*.
-. Select one or more roles to add to this group, then click *Next*.
+. Click menu:image:images/cog.png[][Settings].
```

Comment:
Same as above, suggest changing to:

Click image:images/cog.png[] and select * Settings* .